### PR TITLE
Major Release 2020

### PR DESCRIPTION
**Pull request by kevinkowa**
BREAKING CHANGE: api def tag 'sdk-major-release-2020' gen tag 3.19.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,52 @@
+# [9.0.0-rc.3](https://github.com/watson-developer-cloud/java-sdk/compare/v9.0.0-rc.2...v9.0.0-rc.3) (2020-12-03)
+
+
+### Bug Fixes
+
+* **visual-recognition-v4:** remove trainingStatus as a parameter for create & update Collection ([02e3dfd](https://github.com/watson-developer-cloud/java-sdk/commit/02e3dfd0fb972f3081c91921f6acf61fdcc2f1e6))
+* **visual-recognition-v4:** remove trainingStatus as a parameter for create & update Collection ([5568bf9](https://github.com/watson-developer-cloud/java-sdk/commit/5568bf99e47ad92420253c936db7bc8fac13a7f0))
```
Comment (mediumTaj) [inline CHANGELOG.md:7]: I think these can be removed - I don't think there was ever a release with these in the sdk code

File: MIGRATION-V9.md [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+
+`Features`
+* Changed `metadata` property type from `MetadataOptions` to `Map<String, Object>`
+
+`ListModelsOptions`
+* Remove Builder
+
+### Personality Insights v3 Instance
+
+* No changes
```
Comment (mediumTaj) [inline MIGRATION-V9.md:230]: Added deprecation notice

File: MIGRATION-V9.md [hunk 3]
```diff
@@ -0,0 +1,324 @@
...
+
+### Tone Analyzer v3 Models
+
+* No changes
+
+### Visual Recognition v3 Instance
+
+* No changes
```
Comment (mediumTaj) [inline MIGRATION-V9.md:288]: Added deprecation notice

File: MIGRATION-V9.md [hunk 4]
```diff
@@ -0,0 +1,324 @@
...
+
+### Visual Recognition v3 Models
+
+* No changes
+
+### Visual Recognition v4 Instance
+
+* No changes
```
Comment (mediumTaj) [inline MIGRATION-V9.md:296]: Added deprecation notice

File: MIGRATION-V9.md [hunk 5]
```diff
@@ -0,0 +1,324 @@
...
+
+### Visual Recognition v3 Instance
+
+* No changes
+
+### Visual Recognition v3 Models
+
+* No changes
+
+### Visual Recognition v4 Instance
```
Comment (mediumTaj) [inline MIGRATION-V9.md:294]: These should be called `Service` rather than `Instance`

> Visual Recognition v4 Service

**Review by mediumTaj — DISMISSED** [review]
👍 looks good, just a couple of nit picky things

File: speech-to-text/src/test/java/com/ibm/watson/speech_to_text/v1/SpeechToTextIT.java [hunk 6]
```diff
@@ -23,52 +23,7 @@
...
-import com.ibm.watson.speech_to_text.v1.model.RecognitionJobs;
-import com.ibm.watson.speech_to_text.v1.model.RecognizeOptions;
-import com.ibm.watson.speech_to_text.v1.model.SpeechModel;
-import com.ibm.watson.speech_to_text.v1.model.SpeechModels;
-import com.ibm.watson.speech_to_text.v1.model.SpeechRecognitionResult;
-import com.ibm.watson.speech_to_text.v1.model.SpeechRecognitionResults;
-import com.ibm.watson.speech_to_text.v1.model.Word;
-import com.ibm.watson.speech_to_text.v1.model.WordAlternativeResults;
-import com.ibm.watson.speech_to_text.v1.model.Words;
+import com.ibm.watson.speech_to_text.v1.model.*;
```
[the remark this answers is not in the document]
Comment (kevinkowa) [reply speech-to-text/src/test/java/com/ibm/watson/speech_to_text/v1/SpeechToTextIT.java]: Fixed!

File: CHANGELOG.md (hunk 1 above)
Comment (kevinkowa) [reply CHANGELOG.md]: Fixed!

File: MIGRATION-V9.md (hunk 2 above)
Comment (kevinkowa) [reply MIGRATION-V9.md]: Fixed!

File: MIGRATION-V9.md (hunk 3 above)
Comment (kevinkowa) [reply MIGRATION-V9.md]: Fixed!

File: MIGRATION-V9.md (hunk 4 above)
Comment (kevinkowa) [reply MIGRATION-V9.md]: Fixed!

File: MIGRATION-V9.md (hunk 5 above)
Comment (kevinkowa) [reply MIGRATION-V9.md]: Fixed!

File: MIGRATION-V9.md [hunk 7]
```diff
@@ -0,0 +1,324 @@
...
+
+### Visual Recognition v3 Service
+
+* Added deprecation notice
+
+### Visual Recognition v3 Models
+
+* No changes
+
+### Visual Recognition v4 Service
```
Comment (mediumTaj) [inline MIGRATION-V9.md:294]: something like `getTrainingUsage()`: `startTime` and `endTime` parameter types changed from `String?` to `Date?`

Comment (kevinkowa) [reply MIGRATION-V9.md:294]: is this for v4? I have 
`GetTrainingUsageOptions`
* Changed `startTime` and `endTime` property type from `String` to `Date`

Comment (mediumTaj) [reply MIGRATION-V9.md:294]: gotcha thanks

**Review by mediumTaj — APPROVED** [review]
👍 looks good!

**Pull request merged**